gradlib/gradlib/GemmTuner.py

```diff
@@ -15,6 +15,7 @@
 atol = 1
 
 CACHE_INVALIDATE_BUFFERS = int(os.getenv("CACHE_INVALIDATE_BUFFERS", "37"))
+ONE = torch.ones(1, dtype=torch.float32, device='cuda')
```

Comment:
Let's add this as a class member the `Gemm` class? If I'm not mistaken this will run the moment anything from this file is imported (even if it's not used) which can prematurely initialize the CUDA context.

Specifically: let's initialize the class member to `None`. Then in `__init__`, if this class member is not set, initialize it.

Reply:
Same here

Reply:
Same here about how the PR author should not resolve conversations that are inconvenient but unresolved.